README.md

```diff
@@ -26,9 +26,9 @@
 
 ## Intro
 
-Export your excel VBA project source code for use with Git (or any VCS) from the Excel Developer ribbon.
+Export your excel VBA project source code for use with Git (or any VCS) from the Excel Developer ribbon, a pure VBA add-in for code export, no need to install a COM add-in
```

Comment:
Agreed, to confuse this even more I have added an additional paragraph to the end of the intro :wink: